docs_app/content/tutorial/2-unsubscribing.md

```diff
@@ -0,0 +1,39 @@
+# 2. Unsubscribing
+
+## TLDR:
+
+- Calls to [`subscribe`](API) return a [`Subscription`](API) object.
+- Calling [`unsubscribe`](API) on the [`Subscription`](API) object notifies the [producer](GL) to stop sending values, and triggers the [teardown](GL) of any underlying resources the [producer](GL) has registered for teardown.
+- Failure to [unsubscribe](GL) from asynchronous observables will result in unnecessary resource use, and even memory leaks.
+- Any errors that occur during teardown will be collected and rethrown as an [`UnsubscriptoinError`](API).
+
+## Overview
+
+Just as you need to know how to start getting values from your observable, you must also need to know how to tell the [producer](GL) to stop sending values, and perhaps more importantly, to [teardown](GL) and free up resources.
```

Comment:
Minor: Consider changing this: "you must also need to know" to this "you must also know".